civis/ml/_model.py

```diff
@@ -192,6 +192,18 @@ def _exception_from_logs(exc, job_id, run_id, client, nlog=15):
     return exc
 
 
+def _get_credential(credential, client=None):
+    """Get the credential ID from either an integer or a string."""
```

Comment:
The API will give you back the ID for the credential rather than the name:
```python
In [8]: container = client.scripts.get_containers(6229384)

In [9]: container.arguments
Out[9]:
{'CALIBRATION': None,
 'CIVIS_FILE_ID': 5092501,
 'CVPARAMS': '{}',
 'DB': {},
 'DEBUG': True,
 'EXCLUDE_COLS': 'tsmart_county_fips tsmart_census_block_fips',
 'GITHUB': 5991,
 'IF_EXISTS': 'fail',
 'MODEL': '5092505',
 'OOSDB': {},
 'PARAMS': '{}',
 'PRIMARY_KEY': 'voterbase_id',
 'TABLE_NAME': None,
 'TARGET_COLUMN': 'registered_dem'}
```
I had included this originally so that I could handle the result of this call in `from_existing`. I'll remove this and have `from_existing` convert the ID returned to a string before instantiating the class.